CMakeLists.txt

```diff
@@ -151,6 +151,7 @@ set(PARSEC_MAX_LOCAL_COUNT 20 CACHE STRING "Number of local variables for tasks
 set(PARSEC_MAX_PARAM_COUNT 20 CACHE STRING "Number of parameters for tasks (default 20)")
 set(PARSEC_MAX_DEP_IN_COUNT 10 CACHE STRING "Number of input flows for each task (default 10)")
 set(PARSEC_MAX_DEP_OUT_COUNT 10 CACHE STRING "Number of output flows for each task (default 10)")
+set(PARSEC_MAX_DEVICE_FLOWS ${PARSEC_MAX_PARAM_COUNT} CACHE STRING "Number of parameters for tasks (default: same as PARSEC_MAX_PARAM_COUNT)")
```

Comment:
discussion: 
1. do we really need a second parameter here? 
2. can MAX_DEVICE_FLOWS be smaller than MAX_PARAM_COUNT or is it erroneous?
3. changing the type of MAX_PARAM_COUNT could have some adverse effects on PTG minimal execution context, due to the fragile reliance on alignment in that structure, that would need to be checked if we want to go there

Reply:
I wanted to go with a separate parameter here because `MAX_PARAM_COUNT` is used in many places that I didn't want to touch and where it is not needed for my use case, including the task structure and the communication backend. I didn't want to touch that, party because it blows up all the size of the tasks unnecessarily. But with #694 this issue would be moot and we could expand this to all flows.